src/components/oTextarea.js

```diff
@@ -0,0 +1,247 @@
+import o from 'ojs-core';
+import isValidInputEventName from '../utils/isValidInputEventName';
+import updateComponentDb from "../utils/updateComponentDb";
+
+export default function oTextarea(config) {
+  if (!(this instanceof oTextarea)) {
+    return new oTextarea(config);
+  }
+
+  this.element = o('textarea');
+  this.textarea = this.element.element;
+  this.dbSet = false;
+  this.dbObject = null;
+  this.dbKey = null;
+  this.index = null;
+
+  //TODO: initConfigService
+  if(typeof config === 'object') {
+    //
+  }
+}
+
+oTextarea.prototype.attr = function(attrs) {
+  this.element.attr(attrs);
+  return this;
+}
+
+oTextarea.prototype.classList = function(classList) {
+  this.element.classList(classList);
+  return this;
+}
+
+oTextarea.prototype.cols = function(cols) {
+  if(isNaN(cols)) return this;
+
+  this.element.attr({
+    cols
+  });
+  return this;
+}
+
+oTextarea.prototype.db = function (db, name, updateOn = 'change') {
+  if(!isValidInputEventName(updateOn)) {
+    updateOn = 'change';
+  }
+
+  if(this.dbSet) return this;
+
+  this.dbSet = true;
+  this.dbObject = db;
+  this.dbKey = name;
+
+  this.element.event({
+    name: updateOn,
+    fn: e => {
+      updateComponentDb(
+        this.dbObject,
+        this.dbKey,
+        this.index,
+        e.target.value
+      );
+    }
+  });
+
+  this.textarea.value = this.index ? db[this.index][name] : db[name];
+  return this;
+}
+
+oTextarea.prototype.disabled = function(required) {
+  if(disabled !== 'true' && disabled !== 'false' && typeof disabled !== 'boolean') return this;
```

Comment:
To samo co w oInput. Bym zrobił toLowerCase, żeby można było wpisać też 'True' jak chcemy stringi